content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md

```diff
@@ -33,6 +33,49 @@ You will need to have the following tools installed:
 - `cosign`（[安装指南](https://docs.sigstore.dev/cosign/installation/)）
 - `curl`（通常由你的操作系统提供）
 
+<!-- 
+## Verifying binary signatures
+
+The Kubernetes release process signs all binary artifacts (tarballs, SPDX files,
+standalone binaries) by using cosign's keyless signing. To verify a particular
+binary, retrieve it together with its signature and certificate: -->
+
+## 验证二进制签名 {#verifying-binary-signatures}
+
+Kubernetes 发布过程使用 cosign 的无密钥签名对所有二进制制品（tarballs、SPDX 文件、独立二进制文件）进行了签名。要验证特定的二进制文件，在获取这个二进制文件的时候，可以配套获取签名和证书的信息，参考下面的示例：
+
+```bash
+URL=https://dl.k8s.io/release/v{{< skew currentVersion >}}.0/bin/linux/amd64
+BINARY=kubectl
+
+FILES=(
+    "$BINARY"
+    "$BINARY.sig"
+    "$BINARY.cert"
+)
+
+for FILE in "${FILES[@]}"; do
+    curl -sSfL --retry 3 --retry-delay 3 "$URL/$FILE" -o "$FILE"
+done
+```
+
+<!-- Then verify the blob by using `cosign`: -->
+然后使用 `cosign` 验证 blob：
+
+```shell
+cosign verify-blob "$BINARY" --signature "$BINARY".sig --certificate "$BINARY".cert
+```
+
+{{< note >}}
+<!-- 
+To learn more about keyless signing, please refer to [Keyless
+Signatures](https://github.com/sigstore/cosign/blob/main/KEYLESS.md#keyless-signatures).
+-->
+
+想要进一步了解 `KEYLESS`，请参考
```

Comment:
```suggestion
想要进一步了解无密钥签名，请参考
```

Reply:
Thank you for taking the time to review my pull request and for providing these suggestions. I have made the requested changes to the code . Please let me know if you have any further questions or concerns. Thank you again for your help!